daisy_workflows/image_import/import_disk.wf.json

```diff
@@ -50,7 +50,10 @@
       "CreateInstances": [
         {
           "Name": "inst-importer",
-          "Disks": [{"Source": "disk-importer"}],
+          "Disks": [
+            {"Source": "disk-importer"},
+            {"Source": "${disk_name}"}
```

Comment:
Why is disk attached here instead of in the script?

I've added Andrew to give some context on why we're currently attaching the second disk in the script.

Reply:
My understanding (which I double checked by running the workflow) is that GCP allows you to resize an attached disk. So, moving the attach step here saves on boilerplate from the shell script.

That's useful since the subsequent change will need to attach another disk where the input disk image will be staged. And that disk image may need to be resized as well. 

So, in short, I'm trying to remove boilerplate in preparation for adding some similar code.

Reply:
I actually don't remember it ever being the case that we couldn't resize an attached disk, so I have no idea why this was done this way previously.